packages/interface/src/features/applications/hooks/useCreateApplication.ts

```diff
@@ -18,7 +17,7 @@ export type TUseCreateApplicationReturn = Omit<
 };
 
 export function useCreateApplication(options: {
-  onSuccess: (data: Transaction<string[]>) => void;
+  onSuccess: (hash: `0x${string}`) => void;
```

Comment:
small nit but you could use the Hex type from viem

Reply:
sure